cfg/CFG.h

```diff
@@ -84,6 +95,7 @@ class CFG final {
     core::SymbolRef symbol;
     int maxBasicBlockId = 0;
     int maxRubyBlockId = 0;
+    int maxVariableId = 0;
```

Comment:
Is this value different than the size of the `localVariables` vector?

Reply:
Nope. `maxBasicBlockId` is also `basicBlock.size()` AFAIK.

Reply:
Is it worth using `.size()` instead, to avoid them getting out of sync? `maxBasicBlockId` is different, as basic blocks will be merged and removed from the vector in a later step, so that value doesn't necessarily always reflect the size of the vector.

Reply:
Sure, that's fine with me.

Reply:
I introduced `maxVariableId()` and removed this field.